umake/frameworks/ide.py

```diff
@@ -473,3 +474,83 @@ def post_install(self):
                                                      categories=categories))
         if not self.was_in_arduino_group:
             UI.delayed_display(DisplayMessage(_("You need to logout and login again for your installation to work")))
+            
+class BaseNetBeans(umake.frameworks.baseinstaller.BaseInstaller):
+    """The base for all Netbeans installers."""
+    
+    BASE_URL = "http://download.netbeans.org/netbeans/8.0.2/final/"
+
+    def __init__(self, category, version="8.0.2", flavour=""):
+        #add a separator to the string, like -cpp
+        if flavour != "":
+            flavour = '-' + flavour
+        super().__init__(name="Netbeans",
+                         description=_("Netbeans IDE " + version),
+                         category=category, 
+                         only_on_archs=['i386', 'amd64', 'arm'],
```

Comment:
"arm" doesn't exist on ubuntu, I guess you meant armhf (or maybe armel or both?)